src/saving/FileHelpers.cs

```diff
@@ -18,7 +20,7 @@ public static void MakeSureDirectoryExists(string path)
 
             if (result != Error.AlreadyExists && result != Error.Ok)
             {
-                throw new IOException($"can't create folder: {path}");
+                throw new System.IO.IOException($"can't create folder: {path}");
```

Comment:
Why did you remove the System.IO import?

Reply:
Because System.IO has a File class.
I think, that the Godot.File class will be used more frequently than System.IO.File in this class

Reply:
Okay, that's fine.